cirq/ops/projectors_test.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+import numpy as np
+import pytest
+
+import cirq
+
+
+def test_projector_qubit():
+    zero_projector = cirq.Projector([[1.0, 0.0]])
+    one_projector = cirq.Projector([[0.0, 1.0]])
+
+    np.testing.assert_allclose(cirq.channel(zero_projector),
+                               ([[1.0, 0.0], [0.0, 0.0]],))
+
+    np.testing.assert_allclose(cirq.channel(one_projector),
+                               ([[0.0, 0.0], [0.0, 1.0]],))
+
+
+def test_projector_from_np_array():
+    zero_projector = cirq.Projector(np.array([[1.0, 0.0]]))
+    np.testing.assert_allclose(cirq.channel(zero_projector),
+                               ([[1.0, 0.0], [0.0, 0.0]],))
+
+
+def test_projector_plus():
```

Comment:
Could we throw in one more complicated test ? Maybe one where you need to follow the math instead of just eyeballing it :)

Reply:
Thank you so much for that comment! I had mistakenly used * instead of np.matmul and I was computing the wrong answer. Thanks to you suggestion for a more advanced test, I caught it.

The test is [1, 0] and [1, 1] which should result in the identity projection.